includes/abstracts/abstract.llms.post.model.php

```diff
@@ -44,6 +44,7 @@
  * @since 3.34.0 Add `set_bulk()` method that will allow to update an object at once given an array of properties.
  * @since 3.34.0 Refresh the whole $post property with the just updated instance of WP_Post after updating it.
  * @since 3.36.1 In `set_bulk()` method, use WP_Error::$errors in place of WP_Error::has_errors() to support WordPress version prior to 5.1.
+ * @since [version] Fix quote slashing in `set_bulk()` when the user is not an admin.
```

Comment:
Class-level changelog *used* to contain a duplicate copy of *everything* included in method changelogs. WE decided to stop doing this a while but copying examples from existing classes causes issues because *we didn't open every class and removed duplicate changelogs when we decided to change the rules.

Review the doc guidelines here to see what should be included in a class changelog: https://github.com/gocodebox/lifterlms/blob/trunk/docs/documentation-standards.md#changes-warrating-a-changelog-entry

I am sorry for the schizophrenia of our changelogs... We have a made a lot of changes over the years and looking at *existing* code is not always the best example becaues we simply haven't had the time to prioritize blanket updating of the entire codebase to adhere to current standards

```suggestion
```